napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -131,7 +206,13 @@ def setup_ui(self, enabled=True):
         self.enabled_checkbox.setMinimumSize(QSize(20, 0))
         self.enabled_checkbox.setText("")
         self.row1.addWidget(self.enabled_checkbox)
-        self.plugin_name = QLabel(self)
+        self.plugin_name = QPushButton(self)
+        # Do not want to highlight on hover unless there is a website.
+        if self.url and self.url != 'UNKNOWN' and self.url != '':
```

Comment:
```suggestion
        if self.url and self.url != 'UNKNOWN':
```

`self.url` evaluating to true means that string is not empty